settings/resource_default_namespace_setting.go

```diff
@@ -11,15 +11,15 @@ import (
 var defaultNamespaceSetting = workspaceSetting[settings.DefaultNamespaceSetting]{
 	settingStruct: settings.DefaultNamespaceSetting{},
 	readFunc: func(ctx context.Context, w *databricks.WorkspaceClient, etag string) (*settings.DefaultNamespaceSetting, error) {
-		return w.Settings.ReadDefaultWorkspaceNamespace(ctx, settings.ReadDefaultWorkspaceNamespaceRequest{
+		return w.Settings.GetDefaultNamespaceSetting(ctx, settings.GetDefaultNamespaceSettingRequest{
```

Comment:
@hectorcast-db looks like the latest version of the SDK breaks a bunch of things.

Reply:
cc @mgyucht

Reply:
I would suggest to wait until new SDK is integrated, and then add these settings

Reply:
Got it. When does that happen usually?

Reply:
Hopefully it will be this week as we need new Go SDK for few other things

Reply:
We are doing a couple of fixes to the generation. After it's done (Today or tomorrow latest) we will bump the SDK version.

Reply:
@gauthamsunjay latest Go SDK is merged into main branch, please rebase...